lib/car.rb

```diff
@@ -0,0 +1,23 @@
+#
+#
+#
+
+class Car
+
+  attr_reader :make, :model, :monthly_payment, :loan_length, :total_cost, :color
+
+  def initialize(make_model, monthly_payment, loan_length, color = nil)
+    @make_model_iv = make_model.split
```

Comment:
I probably wouldn't make `make_model_iv` and instance variable since it won't be used anywhere else in the class or need to be accessed later